notebook/analysis/benchmark_results.ipynb

```diff
@@ -0,0 +1,668 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Benchmark Results\n",
+    "This notebook visualizes the output from the different models on different classification problems"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "\n",
+    "import pandas as pd\n",
+    "from plotnine import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sepsis classification"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['../../results/single_label.sepsis.PCA.pytorch_supervised.refinebio_labeled_dataset.3.tsv', '../../results/single_label.sepsis.PCA.pytorch_supervised.refinebio_labeled_dataset.2.tsv', '../../results/single_label.sepsis.logistic_regression.refinebio_labeled_dataset.1.tsv', '../../results/single_label.sepsis.PCA.logistic_regression.refinebio_labeled_dataset.1.tsv', '../../results/single_label.sepsis.PCA.logistic_regression.refinebio_labeled_dataset.4.tsv']\n"
```

Comment:
Do all of the PCA preprocessed datasets here use the top 100 components? Do you have any plans to try other numbers of components?

Reply:
Yep, all the models use the same number of PCA components. I want to try other numbers of components/model architectures, etc, but I haven't come up with a good plan for hyperparameter optimization yet.